slack_sdk/socket_mode/aiohttp/__init__.py

```diff
@@ -185,6 +187,13 @@ async def receive_messages(self) -> None:
                 else:
                     await asyncio.sleep(consecutive_error_count)
 
+    async def is_connected(self) -> bool:
```

Comment:
For easier access to the state and better alignment with the sync SocketModeClient, I've added this flag.